client/my-sites/checkout/composite-checkout/test/composite-checkout.js

```diff
@@ -632,8 +632,8 @@ describe( 'CompositeCheckout', () => {
 				container
 			);
 		} );
-		const { getByLabelText } = renderResult;
-		expect( getByLabelText( 'Domain Mapping: bar.com' ) ).toBeInTheDocument();
+		const { getByText } = renderResult;
+		expect( getByText( 'Domain Mapping: billed yearly' ) ).toBeInTheDocument();
```

Comment:
Does the page display the domain name for the mapping product at all? If so, can we add an `expect` for that also, just to be sure that the right domain name is being added. (For example, if we introduced a bug which caused a a domain mapping for an empty string to be added, this test would still pass).